extensions/resteasy-classic/rest-client-config/runtime/src/test/java/io/quarkus/restclient/config/RestClientConfigTest.java

```diff
@@ -51,7 +51,7 @@ private void verifyConfig(RestClientConfig config) {
         assertThat(config.connectionTTL.get()).isEqualTo(30000);
         assertThat(config.connectionPoolSize).isPresent();
         assertThat(config.connectionPoolSize.get()).isEqualTo(10);
-        assertThat(config.multipart.maxChunkSize.get()).isEqualTo(1024);
```

Comment:
should we consider this as a breaking change?

Reply:
There isn't anything breaking in this chance for now. We just add a new property that takes precedence over the old one.

Reply:
Applied